assembl/static2/js/app/components/search.jsx

```diff
@@ -86,6 +87,14 @@ function getPostUrl(ideaId, postId, phaseIdentifier, slug) {
       questionIndex: 1,
       element: postBase64id
     });
+  } else if (phaseIdentifier === 'thread' && extractId) {
```

Comment:
You shouldn't have `phaseIdentifier === 'thread'` here.

Reply:
PHASES.thread